src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/ArtifactsSizeTest.cs

```diff
@@ -52,15 +51,12 @@ public void CompareArtifactsToBaseline()
         IEnumerable<TarEntry> artifactsTarEntries = Utilities.GetTarballContent(Config.SourceBuiltArtifactsPath).Where(entry => entry.EntryType == TarEntryType.RegularFile);
         IEnumerable<TarEntry> sdkTarEntries = Utilities.GetTarballContent(Config.SdkTarballPath).Where(entry => entry.EntryType == TarEntryType.RegularFile);
 
+        Dictionary<string, int> fileNameCountMap = new Dictionary<string, int>();
         (string FilePath, long Bytes)[] tarEntries = sdkTarEntries.Concat(artifactsTarEntries)
             .Select(entry =>
             {
-                string result = BaselineHelper.RemoveVersions(entry.Name);
-                result = BaselineHelper.RemoveRids(result);
-                result = BaselineHelper.RemoveNetTfmPaths(result);
```

Comment:
Was there a reason RemoveNetTfmPaths was removed?  This did a nicer job of replacing TFMs than the current logic.  net7.0 should be netx.y not netx.y.z.  It may be this needs to be done before replacing versions.

Reply:
I noticed that RemoveVersions uses ```return RemoveNetTmfPath(result)```, so calling 
```
result = BaselineHelper.RemoveRids(result);
result = BaselineHelper.RemoveNetTfmPaths(result);
```
in my code was redundant. Keeping netx.y in mind, I'll add the call to RemoveNetTmfPath back in before a call to RemoveVersions.

Reply:
In that case, I think RemoveVersions will need to be refactored so that it calls RemoveNetTfmPaths before replacing the version strings.

Reply:
[Refactored RemoveVersions](https://github.com/dotnet/installer/pull/17233/commits/d74d7bac55d9c37ee43694f72df4c2407218046a#diff-2ed96e14569dd53766d8c841db49d63662b897532bc8e1bb9837f87e52476f12R105) by moving call to RemoveNetTfmPAths before replacing the version strings.